src/main/resources/com/leinardi/pycharm/mypy/MypyBundle.properties

```diff
@@ -50,6 +50,7 @@ plugin.notification.unable-to-run-mypy.subtitle=Unable to run Mypy
 plugin.notification.unable-to-run-mypy.content=Mypy is installed inside the project environment but the plugin \
   is not able to run it. If you just installed it try to run File -> Synchronize or restart your IDE. \
   If the problem persists you may need to manually enter the path to the Mypy executable inside the Plugin settings.
+plugin.notification.abnormal-exit.subtitle=Mypy exited abormally
```

Comment:
There was a typo on the word `abnormally` (`abormally`). I fixed it and already pushed on master.